packages/devextreme/js/__internal/scheduler/appointment_popup/appointment_popup.test.ts

```diff
@@ -1339,6 +1339,28 @@ describe('Appointment Popup Form', () => {
     });
   });
 
+  describe('Form customization', () => {
+    it('should propagate editing.form options to the form instance', async () => {
```

Comment:
I suggest to add a test that if user's custom form options has `elementAttr.id="someValue"`, then the form has correct values for both 'elementAttr.id' and 'elementAttr.class' (this is a default property) values.